app/components/person/search-form.js

```diff
@@ -41,6 +41,30 @@ export default class SharedPersoonPersoonSearchFormComponent extends Component {
     this.personen = A();
   }
 
+  @action
+  setVoornaam(event) {
+    if (event && typeof event.preventDefault === 'function')
+      event.preventDefault();
+    this.voornaam = event.target.value.trim();
+    this.search.perform();
+  }
+
+  @action
+  setAchternaam(event) {
+    if (event && typeof event.preventDefault === 'function')
+      event.preventDefault();
+    this.achternaam = event.target.value.trim();
+    this.search.perform();
+  }
+
+  @action
+  setRrn(event) {
```

Comment:
I would still call the method `setRijksregisternummer`. What Rrn means can be derived from context, but usually names in full hold up better for readability.